### PR TITLE
fix: don't fetch old values when changing data

### PR DESCRIPTION
**Pull request by Sparkier**

### CODE REVIEW

**Comment by Sparkier** [comment]
This still has a problem where sometimes metrics can't be written to the table directly and it has some delay. I did not get to fully debugging anymore. But this is still strictly better than what we had before.

File: src/routes/api/metric/+server.ts [hunk 1]
```diff
@@ -24,22 +24,35 @@ export async function POST({ locals: { supabase, getSession }, request }) {
 		error(500, 'Invalid project data');
 	}
 
-	const fetchRes = await supabase
-		.from('metrics')
-		.select('id, prediction_id, metric_name, metric')
-		.eq('prediction_id', prediction.id);
+	const clear = requestData.clear as boolean;
+
+	console.log('id: ', prediction.id);
```
Comment (cabreraalex) [inline src/routes/api/metric/+server.ts:29]: nit: console log

File: src/routes/api/metric/+server.ts [hunk 2]
```diff
@@ -24,22 +24,35 @@ export async function POST({ locals: { supabase, getSession }, request }) {
...
 
-	if (fetchRes.data && fetchRes.data.length > 0) {
-		return json(fetchRes.data[0]);
+		if (fetchRes.data && fetchRes.data.length > 0) {
+			return json(fetchRes.data[0]);
+		}
 	}
 
 	let metric: number;
+	console.log('label: ', label);
```
Comment (cabreraalex) [inline src/routes/api/metric/+server.ts:44]: more

**Review by cabreraalex — APPROVED** [review]
Can you remove console logs and add a high priority linear issue for what's left?

**Comment by Sparkier** [comment]
Seems like we have a rule that prevents this merge.

**Pull request merged**